### PR TITLE
PLAT-697: fix managing of pending counters

### PR DESCRIPTION
**Pull request by ivanshibitov**
1. remove pending counters 
2. use known table instead of 1.
3. update tests

### CODE REVIEW

File: ledger-core/virtual/execute/execute.go [hunk 1]
```diff
@@ -540,7 +540,7 @@ func (s *SMExecute) stepStartRequestProcessing(ctx smachine.ExecutionContext) sm
 			panic(throw.Impossible())
 		}
 
-		state.IncrementPotentialPendingCounter(s.execution.Isolation)
+		//state.IncrementPotentialPendingCounter(s.execution.Isolation)
```
Comment (sexyhamster) [inline ledger-core/virtual/execute/execute.go]: Delete please

File: ledger-core/virtual/callregistry/working_table.go [hunk 2]
```diff
@@ -31,12 +31,23 @@ func NewWorkingTable() WorkingTable {
...
 		return wt.requests[flag-1]
 	}
 	panic(throw.IllegalValue())
 }
 
+func (wt WorkingTable) checkInterferenceFlag(flag contract.InterferenceFlag) bool {
+	return flag > 0 && int(flag) <= len(wt.requests)
+}
+
+func (wt WorkingTable) CountActive(flag contract.InterferenceFlag) int {
```
Comment (sexyhamster) [inline ledger-core/virtual/callregistry/working_table.go:44]: I wonder why you can't use table.GetList(flag).CountActive? This is literally the same

Comment (ivanshibitov) [reply ledger-core/virtual/callregistry/working_table.go:44]: It's with checks and shorter to use

Comment (sexyhamster) [reply ledger-core/virtual/callregistry/working_table.go:44]: The checks are exactly the same, and I prefer to have only one way of using this lists. Please do not create duplicate functions

Comment (ivanshibitov) [reply ledger-core/virtual/callregistry/working_table.go:44]: ok, GetList also has checks.
But, it's still shorter

Comment (sexyhamster) [reply ledger-core/virtual/callregistry/working_table.go:44]: Let's avoid creatung duplicates when it is not required.

Comment (bigbes) [reply ledger-core/virtual/callregistry/working_table.go:44]: overall I agree with @sexyhamster, you save 10 charachters in 2 places of code, but you've added duplicate code with length > 20 ¯\_(ツ)_/¯

Comment (ivanshibitov) [reply ledger-core/virtual/callregistry/working_table.go:44]: It's used more than 30 times is code. It's not 10 charachters.

Comment (sexyhamster) [reply ledger-core/virtual/callregistry/working_table.go:44]: Half of them are tests for new function.

Comment (ivanshibitov) [reply ledger-core/virtual/callregistry/working_table.go:44]: negotiated

**Pull request merged**